diff --git a/node_modules/expo-router/build/react-navigation/core/getStateFromPath.js b/node_modules/expo-router/build/react-navigation/core/getStateFromPath.js
--- a/node_modules/expo-router/build/react-navigation/core/getStateFromPath.js
+++ b/node_modules/expo-router/build/react-navigation/core/getStateFromPath.js
@@ -41 +41 @@
-const queryString = __importStar(require("query-string"));
+const queryString = __importDefault(require("query-string")).default;
